web-api/src/lambdaWrapper.js

```diff
@@ -10,8 +10,10 @@ exports.headerOverride = {
   'X-Content-Type-Options': 'nosniff',
 };
 
-exports.lambdaWrapper = lambda => {
+exports.lambdaWrapper = (lambda, options = {}) => {
   return async (req, res) => {
+    const shouldMimicApiGateway =
```

Comment:
`shouldMimicApiGateway` seems very generic.  In which way should it mimic the real AWS API Gateway?  In this case it wants to mimic the async-ness of it.  I would call it for what it is: something like `isAsyncNotInProduction`.

.... Oh wait.  this `NODE_ENV` environment variable.  That isn't the environment we deployed to, right?  I.e. not `exp3`, `test`, or `staging`.

Reply:
I renamed it to `shouldMimicApiGatewayAyncEndpoint` which I think is more descriptive I hope?  yeah, NODE_ENV will only be 'production' when it's deployed to a real AWS environment